### ISSUE TITLE
training loss curve of the pretrain stage

### ISSUE THREAD

**Issue by guozhiyao** [JiuhaiChen/BLIP3o#16]
Could you please share the training loss curve of the pretrain stage?

**Comment by JiuhaiChen** [comment]
Hi! Here’s the pretraining curve for the 4B model. Because we resumed training several times, the plot is actually made up of multiple runs.
https://wandb.ai/jchen169/huggingface/reports/Qwen-VL-3B-Diffusion-transformer-1B--VmlldzoxMjg1NDA1MQ

**Comment by guozhiyao** [comment]
> Hi! Here’s the pretraining curve for the 4B model. Because we resumed training several times, the plot is actually made up of multiple runs. https://wandb.ai/jchen169/huggingface/reports/Qwen-VL-3B-Diffusion-transformer-1B--VmlldzoxMjg1NDA1MQ

@JiuhaiChen thinks a lot, could you share the 8b model?

**Comment by JiuhaiChen** [comment]
<img width="1105" alt="Image" src="https://github.com/user-attachments/assets/fda843e0-ade9-4eb0-bc92-90aed9a68032" />

Hi, this is 8B with 1e-4 constant learning rate.

**Comment by guozhiyao** [comment]
@JiuhaiChen thanks a lot. I tried to reproduce this performance, also using an 8b model, 32 cards, batch size16 lr1e-4, the initial loss was around 6.5, and it dropped to around 3 after 600 rounds. Is this normal?

**Comment by JiuhaiChen** [comment]
I recommend training for a longer to monitor the loss, we feel that training loss does not reflect the image generation quality. A more reliable metric is the gradient norm: just ensure there are no sudden spikes for the grad norm value.

**Comment by guozhiyao** [comment]
@JiuhaiChen thanks a lot.

**Comment by JiuhaiChen** [comment]
Hi, this is the SFT stage, i put the wandb link, lmk if you cannot open it.
https://api.wandb.ai/links/jchen169/oa6sfhzw

**Comment by whycantfindaname** [comment]
> Hi, this is the SFT stage, i put the wandb link, lmk if you cannot open it. https://api.wandb.ai/links/jchen169/oa6sfhzw

lHello, thank you for your wonderful work and responses. I noticed that the currently provided one is the training loss curve of the pretrain stage. I wonder if you can provide the loss curve of the SFT stage.

**Comment by JiuhaiChen** [comment]
Hi, we didn’t log into wandb for SFT stage, but you can plot the loss in the trainer_state.json, let me know if you have any questions.
